docs/docs/60-new-docs/50-user-guide/20-how-to-guides/15-working-with-freight.md

```diff
@@ -110,7 +110,22 @@ sometimes wish to override that alias with one of their own choosing. This can
 make it easier to identify a particularly important (or problematic) `Freight`
 resource as it progresses through the `Stage`s of a pipeline.
 
-This is conveniently accomplished via the Kargo CLI:
+To update the `Freight` Alias:
+
+<Tabs groupId="update-alias">
+<TabItem value="ui" label="Using the UI" default>
+
+1. Click the three dots on the `Freight` in the <Hlt>Freight Timeline</Hlt>,
+    then select <Hlt>Change Alias</Hlt>:
+
+    ![update-freight-alias](img/freight-alias.png)
+1. Update Alias and submit it.
+
+    ![update-freight-alias](img/freight-alias-2.png)
+
```

Comment:
```suggestion
1. Click the three dots on the `Freight` in the <Hlt>Freight Timeline</Hlt>,
   then select <Hlt>Change Alias</Hlt>:

   ![update-freight-alias](img/freight-alias.png)
1. Update the Alias and submit it.

   ![update-freight-alias](img/freight-alias-2.png)

```

Reply:
(Indentation and "the Alias")

Reply:
With Markdown, and while working with lists, you typically ensure the new line of the same item starts at the point where the first sentence minus the bullet point starts.

For numbered lists, this is `number + dot + space = 3`.
For bulleted lists, this is `bullet + space = 2`.